### PR TITLE
added part b to readme

### LINKED ISSUE #2: Part B of Readme
A description of which features you used in training your model, and how your retrained the model performs better than the baseline model

### PR DESCRIPTION
**Pull request by AlexanderChiuy**
resolves #2 

Still needs some work on the description of attributes and why our model is better than the baseline.

### CODE REVIEW

File: model_build.ipynb [hunk 1]
```diff
@@ -2821,7 +2821,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.7"
+   "version": "3.8.3"
```
Comment (joycewu37) [inline model_build.ipynb:2824]: is this change necessary for the documentation PR?

File: .ipynb_checkpoints/model_build-checkpoint.ipynb [hunk 2]
```diff
@@ -0,0 +1,2829 @@
+{
```
Comment (joycewu37) [inline .ipynb_checkpoints/model_build-checkpoint.ipynb:1]: is this file part of the documentation?

**Review by joycewu37 — COMMENTED** [review]
just making sure the files changed in the PR are directly affecting part b of the readme

**Comment by michellejli77** [comment]
Not updated
